articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,163 @@
+---
+title: How to create a availability set | UKCloud Ltd
+description: Creating a availability set in UKCloud for Microsoft Azure
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: 
+toc_sub3:
+toc_sub4:
+toc_title: Create a availability set
+toc_fullpath: Users/How To/azs-how-create-availability-set.md
+toc_mdlink: azs-how-create-availability-set.md
+---
+
+# Overview
+
+This guide will cover how to increase the availability and reliability of your virtual machines on Azure Stack using **availability sets**. availability sets ensure that VMs you deploy on Azure Stack are evenly distributed across multiple physical nodes in a cluster. Doing this ensure that, if a hardware or software failure happens within Azure Stack, only a subset of your VMs are impacted and your overall solution remains available and operational.
```

Comment:
Doing this ensures that (ensure => ensures)

Reply:
outdated now